### DISCUSSION TITLE
How to re-open bug report closed in error?

### DISCUSSION THREAD

**Discussion by klossner** [mdn/mdn-community#531] [Content] [answered]
Bug report `https://github.com/mdn/content/issues/26262` (a documentation problem) was marked closed but the bug was not actually fixed. How can this bug be moved back to open state? I'm the author and I don't see a control to let me change state.

**Comment by hamishwillee** [comment, marked as answer]
Only maintainers can reopen bugs. You did the right thing adding a comment to the original bug to highlight a (potential) problem with the fix. 

I haven't reopened it because the bug is fixed - the site just hasn't been rebuilt yet.
